flycheck.el

```diff
@@ -7518,6 +7513,14 @@ See URL `https://github.com/eslint/eslint'."
                                    (flycheck-error-message err))))
                           (flycheck-sanitize-errors errors))
                   errors)
+  :enabled (lambda (checker)
+             (let* ((directory (concat
+                                (flycheck-compute-working-directory checker)
+                                "."))
```

Comment:
Will that work if directory does not exist yet? When you open file in Emacs <project-dir>/components/some_component.js and components directory does not exists yet.